sources/s63/sections/00-glossary.adoc

```diff
@@ -0,0 +1,122 @@
+== Glossary
+
+=== Glossary of S-63 Data Protection Scheme Terms
+
+
+
+==== Blowfish
+
+Encryption algorithm used by the protection scheme
+
+==== Cell Key
+
+Key used to produce encrypted ENC, and required to decrypt the encrypted ENC information.
+
+==== Cell Permit
+
+Encrypted form of Cell key, created specifically for a particular user.
+
+==== Data Client
+
+Term used to represent an end-user receiving the encrypted ENC information. The Data Client will be using a software application (e.g. ECDIS) to perform many of the operations detailed within the scheme. Typically, an ECDIS user.
+
+==== Data Server
+
+Term used to represent an organisation producing encrypted ENCs or issuing Cell Permits to end-users.
+
+==== M_ID
+
+The unique identifier assigned by the SA to each manufacture. Data Servers use this to identify which M_KEY to use when decrypting the userpermit.
+
+==== M_KEY
+
+ECDIS manufacturer's unique identification key provided by the Scheme Administrator to the OEM. It is used by OEMs to encrypt the HW_ID when creating a userpermit.
+
+==== HW_ID
+
+The unique identifier assigned by an OEM to each implementation of their system. This value is encrypted using the OEM's unique M_KEY and supplied to the data client as a userpermit. This method allows data clients to purchase licences to decrypt ENC cells.
+
+==== SA
+
+Scheme Administrator
+
+==== SHA-1
+
+Secure Hash Algorithm [3]
```

Comment:
@anermina once the references are converted can you change [3] to a `[.source]` block that refers to [3]? Thanks.